Formula/volta.rb

```diff
@@ -0,0 +1,27 @@
+class Volta < Formula
+  desc "Volta: The JavaScript Launcher"
+  homepage "https://volta.sh"
+  url "https://github.com/volta-cli/volta.git", :tag => "v0.7.2"
+
+  depends_on "rust" => :build
+
+  conflicts_with "hadoop", :because => "both install `yarn` binaries"
+  conflicts_with "node", :because => "both install `node` binaries"
+  conflicts_with "npm", :because => "both install `npm` binaries"
+  conflicts_with "yarn", :because => "both install `yarn` binaries"
+
+  def install
+    ENV["VOLTA_HOME"] = prefix.to_s
+    system "./dev/unix/volta-install.sh", "--release", "--skip-setup"
+    system "chmod +x #{opt_prefix}/bin/volta"
+  end
+
+  def caveats; <<~EOS
+    You need to run `volta setup` or add `~/.volta/bin` to your $PATH before using `volta`
+  EOS
```

Comment:
This needs to be in documentation instead of a caveat.

Reply:
Hi @SMillerDev, do you mean on the project documentation or is there a `documentation` block I should use that I've missed somewhere?

If you mean on the project documentation, it's already there. Is there a way I can add a reminder after the formula installation so the user remembers to run this?

Reply:
Caveats are only there for homebrew specific instructions. It doesn't serve to remind users of things.